packages/frontend/src/api.tsx

```diff
@@ -0,0 +1,70 @@
+export interface LoginForm {
+  email: string
+  password: string
+}
+
+export async function logIn(credentials: LoginForm) {
+  return betterFetch<LoginForm>(
+    'POST',
+    'http://localhost:3000/api/auth/login',
+    {},
+    credentials,
+  )
+}
+
+export async function logOut() {
+  return betterFetch('GET', 'http://localhost:3000/api/auth/logout')
+}
+
+export async function getUser(token: string) {
+  try {
+    const payload = await betterFetch(
+      'GET',
+      'http://localhost:3000/api/auth/whoami',
+      { Cookie: `token=${token}` },
+    )
+    return payload
```

Comment:
```suggestion
    return payload.user
```

Since only property is `user`? We can then change `user.user` in homepage (`app/page`) to just `user`.

Reply:
I changed it on the backend, now the payload contains `{ id: 'admin' }`.